cmd/cca/create.go

```diff
@@ -26,14 +27,16 @@ func NewCreateCmd(fs afero.Fs) *cobra.Command {
 		Short: "create a CCA attestation token from the supplied claims and keys",
 		Long: `Create a CCA attestation token from the JSON-encoded claims and
```

Comment:
do we want to surface `-I` in the long description, or should we leave it as a stealth feature?

Reply:
The flag is already listed in the help, so it's not exactly a "stealth" feature. I don't think there is a need to complicate the top-level explanation by mentioning it -- it is a niche use case, and it's usage is obivous (it's a boolean flag that doesn't interact with any other opations).